modal/editModal.html

```diff
@@ -15,27 +15,27 @@ <h5 class="modal-title">Edit Contact</h5>
                     <form id="edit-form" class="row g-3">
                         <div class="col-md-4" data-contact-key="FirstName" data-contact-target="#edit-first-name">
                             <label for="edit-first-name" class="form-label">First Name</label>
-                            <input name="FirstName" type="text" class="form-control" id="edit-first-name">
+                            <input name="FirstName" type="text" class="form-control" id="edit-first-name" maxlength="50">
                         </div>
                         <div class="col-md-4" data-contact-key="LastName" data-contact-target="#edit-last-name">
                             <label for="edit-last-name" class="form-label">Last Name</label>
-                            <input name="LastName" type="text" class="form-control" id="edit-last-name">
+                            <input name="LastName" type="text" class="form-control" id="edit-last-name" maxlength="50">
                         </div>
                         <div class="col-md-4" data-contact-key="PhoneNumber" data-contact-target="#edit-phone-number">
                             <label for="edit-phone-number" class="form-label">Phone Number</label>
-                            <input name="PhoneNumber" type="text" class="form-control" id="edit-phone-number">
+                            <input name="PhoneNumber" type="text" class="form-control" id="edit-phone-number" maxlength="50">
                         </div>
                         <div class="col-8" data-contact-key="Address" data-contact-target="#edit-address">
                             <label for="edit-address" class="form-label">Address</label>
-                            <input name="Address" type="text" class="form-control" id="edit-address" placeholder="1234 Main St">
+                            <input name="Address" type="text" class="form-control" id="edit-address"  maxlength="50">
```

Comment:
Maybe the address can be a bit longer, 50 chars seems a limiting imo

Reply:
Agreed. The longest street name in the USA is 34 characters long, so we should probably double that to make room for apt, floor, etc?